src/matlab2tikz.m

```diff
@@ -3678,11 +3677,14 @@ case size(cData,2) % specific y-data is given
 
     arrowOpts = opts_new();
     if showArrowHead
-        arrowOpts = opts_add(arrowOpts, '->');
+        arrowOpts = opts_add(arrowOpts, '-Straight Barb');
```

Comment:
I think we should warn the user that he should load the TikZ library `arrows.meta`, but I'd have to look up how we do this with other dynamic dependencies to give a snippet of code for this.